packages/flutter/lib/src/gestures/arena.dart

```diff
@@ -64,23 +68,27 @@ class _GestureArenaState {
 class GestureArena {
   final Map<Object, _GestureArenaState> _arenas = new Map<Object, _GestureArenaState>();
 
-  GestureArenaEntry add(Object key, GestureArenaMember member) {
-    _GestureArenaState state = _arenas.putIfAbsent(key, () => new _GestureArenaState());
+  /// Adds a new member to the arena.
```

Comment:
I'd say something like "a new member (i.e. a gesture recognizer) to...".

I'm wary of us writing documentation that someone could have written just by reading the method signature. It can be almost more frustrating than no documentation at all.